waffle/models.py

```diff
@@ -119,49 +119,93 @@ class Flag(BaseModel):
+        max_length=100,
+        unique=True,
+        help_text=_('The human/computer readable name.'),
+        verbose_name=_('Name'),
+    )
+    everyone = models.NullBooleanField(
+        blank=True,
+        help_text=_(
+            'Flip this flag on (Yes) or off (No) for everyone, overriding all '
+            'other settings. Leave as Unknown to use normally.'),
+        verbose_name=_('Everyone'),
+    )
     percent = models.DecimalField(
-        max_digits=3, decimal_places=1, null=True, blank=True,
+        max_digits=3,
+        decimal_places=1,
+        null=True,
+        blank=True,
         help_text=_('A number between 0.0 and 99.9 to indicate a percentage of '
-                    'users for whom this flag will be active.')
+                    'users for whom this flag will be active.'),
+        verbose_name=_('Percent'),
+    )
+    testing = models.BooleanField(
+        default=False,
+        help_text=_('Allow this flag to be set for a session for user testing'),
+        verbose_name=_('Testing'),
+    )
+    superusers = models.BooleanField(
+        default=True,
+        help_text=_('Flag always active for superusers?'),
+        verbose_name=_('Superusers'),
+    )
+    staff = models.BooleanField(
+        default=False,
+        help_text=_('Flag always active for staff?'),
+        verbose_name=_('Staff'),
+    )
+    authenticated = models.BooleanField(
+        default=False,
+        help_text=_('Flag always active for authenticated users?'),
+        verbose_name=_('Authenticated'),
+    )
+    languages = models.TextField(
+        blank=True,
+        default='',
+        help_text=_('Activate this flag for users with one of these languages (comma-separated list)'),
+        verbose_name=_('Languages'),
     )
-    testing = models.BooleanField(default=False, help_text=_(
-        'Allow this flag to be set for a session for user testing.'))
-    superusers = models.BooleanField(default=True, help_text=_(
-        'Flag always active for superusers?'))
-    staff = models.BooleanField(default=False, help_text=_(
-        'Flag always active for staff?'))
-    authenticated = models.BooleanField(default=False, help_text=_(
-        'Flag always active for authenticate users?'))
-    languages = models.TextField(blank=True, default='', help_text=_(
-        'Activate this flag for users with one of these languages (comma '
-        'separated list)'))
     groups = models.ManyToManyField(Group, blank=True, help_text=_(
         'Activate this flag for these user groups.'))
     users = models.ManyToManyField(
-        settings.AUTH_USER_MODEL, blank=True,
-        help_text=_('Activate this flag for these users.')
+        settings.AUTH_USER_MODEL,
+        blank=True,
+        help_text=_('Activate this flag for these users.'),
+        verbose_name=_('Users'),
+    )
+    rollout = models.BooleanField(
+        default=False,
+        help_text=('Activate roll-out mode?'),
```

Comment:
`_` is missing here. Looks like this is the only missing piece.

Reply:
Fixed.